scripts/kind-e2e/e2e.sh

```diff
@@ -3,6 +3,8 @@ set -em
 
 source $(git rev-parse --show-toplevel)/scripts/lib/debug_functions
 
+deploytool=$5
```

Comment:
I already introduced a variable `deploy_operator` so you can use that, e.g.
`if [ "$deploy_operator" = true ]; then`